security/pkg/pki/testdata/crl/cert.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
```

Comment:
When the certs expire, I always find it difficult to understand what to rerun in many of our existing tests. It would be nice to add a comment somewhere to clarify this..